LondonBritishLibrary/orient/BLorient2083.xml

```diff
@@ -0,0 +1,2470 @@
+                                </item> 
+                                <item xml:id="a4">
+                                    <locus from="218ra" to="218vb"/>
+                                    <desc type="Supplication">Benediction</desc>
+                                    <q xml:lang="gez"><title>ጸሎተ፡ ቡራኬ፡</title> <gap reason="ellipsis"/></q>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#1ra"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="de">Gebra Hemamat angekauft fuer 16 Maria Theresia Thaler 
+                                        <date when="1870-08-08">d. 8 August 1870</date>. 
+                                        <persName role="owner" ref="PRS14422FMayer">F. Mayer</persName></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#48va"/>
+                                    <desc type="OwnershipNote">A note in the margin indicating, that this manuscript belonged to the church of
+                                        <placeName ref="INS000000000000000000">Babaḥa ʾAbbo</placeName>. A person with the name
+                                        <persName ref="PRS14421WaldaHanna">Walda Ḥannā</persName>, who was possibly a priest of this 
+                                        church, is also mentioned.</desc> 
+                                    <q xml:lang="gez">ዝመጽሐፍ፡ ዘደብረ፡ በበሐ፡ አቦ፡</q>
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#122vb"/>
+                                    <desc type="OwnershipNote"><persName role="owner" ref="PRS14419Maryamawit">Māryāmāwit</persName>
+                                        is named as an owner, together with her son <persName ref="PRS14420TaklaHaymanot">Takla Hāymānot</persName>.</desc> 
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding xml:id="binding">
+                                <decoNote type="bindingMaterial" xml:id="b1"><material key="wood"/></decoNote>
+                                <decoNote xml:id="b2" type="Boards">Repaired in <placeName ref="INS00001BL">British Museum</placeName>.</decoNote>
+                                <decoNote xml:id="b3" type="Cover"><material key="leather">Blind-tooled leather.</material></decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin><origDate notBefore="1694" notAfter="1706">Between 1694 and 1706</origDate></origin>
```

Comment:
Could you briefly give here the reason for the dating? I assume it refers to the mention of king Iyasu, but from how I understand the description, he is mentioned in an addition, so theoretically the manuscript could also have been produced earlier

Reply:
Me too, I wondered, why Strelcyn was that precise in dating the manuscript. Your idea seems to be right. I surely is the combination of ʾIyāsu I with several church dignataries in a3, that made him think, that this manuscript was written in this time frame. His judgement on the palaeography also supported this dating. However, from the catalogue, it is not clear, whether the note on 217ra, that I encoded as a3, was written down by the same or a similar contemporary hand. 

![Screenshot 2024-05-07 182606](https://github.com/BetaMasaheft/Manuscripts/assets/40291787/cd7d3a17-5689-40d5-a00a-15fcfa2a5d72)

No. 1 in the following list of remarks says, that this benediction was written by a later hand. Does this mean, that the other notes mentioned in remark no. 5 are by a contemporary or by the same hand? If it were the same hand, is the encoding as an addition correct? I think, there is something, that made Strelcyn think, that these addition (a1 and a3) were made around the same time as the main text, unless he does not say explicitely. 


![Screenshot 2024-05-07 182502](https://github.com/BetaMasaheft/Manuscripts/assets/40291787/1b7f15dd-d5d1-471a-92f0-8c6976050edc)

I think it is fair to date the manuscript 1694 to 1706 and pass the responsibility to St. Strelcyn.

Reply:
Without knowing the idiosyncrasies of this catalogue, my guess would be that his note 5 does not refer to an additional note, but to supplications in the main text in which these persons are named. But this is not stated explicitly. I would however change note a3 into an e note, removing the type Record.